Management/src/ActuatorApi/README.md

```diff
@@ -0,0 +1,118 @@
+﻿# Steeltoe Management Sample - Actuators, Administrative Tasks, Metrics and Tracing
+
+ActuatorWeb and ActuatorAPI form an ASP.NET Core-powered sample application that demonstrates how to use several
+Steeltoe libraries on their own and with additional tools
+
+In order to avoid duplicating a significant amount of content, the [ActuatorWeb Readme](../ActuatorWeb/README.md)
+contains the shared information and this document only holds content unique to ActuatorApi.
+
+## Running locally
+
+Be sure to follow the [ActuatorWeb Readme](../ActuatorWeb/README.md) to the point where that application is running
+before continuing here.
+
+### MySQL
+
+This application depends on a MySQL database. Refer to [Common Tasks](../../../CommonTasks.md#MySQL) to start a server
+that matches the credentials defined in [appsettings.Development.json](./appsettings.Development.json).
+
+### Administrative Tasks (initialize the database)
+
+In order to demonstrate [Steeltoe Management Tasks](https://docs.steeltoe.io/api/v3/management/tasks.html), the database
+schema and its contents are managed as administrative tasks.
+
+1. Apply Entity Framework database schema migration scripts:
+
+    ```shell
+    dotnet run --runtask=MigrateDatabase
+    ```
+
+1. Run [`ForecastTask`](./AdminTasks/ForecastTask.cs) to predict weather for the next 7 days:
+
+    ```shell
+    dotnet run --runtask=ForecastWeather
+    ```
+
+    *Optional* - Add forecast data starting from specific date and/or number of days:
+
+    ```shell
+    dotnet run --runtask=ForecastWeather --fromDate=10/18/2024 --days=30
+    ```
+
+    > [!NOTE]  
+    > Use values formatted for invariant culture (`MM/dd/yyyy`) for the `fromDate` parameter.
```

Comment:
I think since we're spelling out exactly what is expected it should be easy enough to work with, and there's less code to deal with since it's .NET defaults. I wouldn't block the change in the future, but I don't see the need to do it now

Reply:
The code doesn't need to change, `yyyy-dd-mm` is already accepted when using the invariant culture,

Reply:
```suggestion
    > For the `fromDate` parameter, use values formatted as `yyyy-dd-mm` or `MM/dd/yyyy`.
```
How about this?